tools/stability-checker/README.md

```diff
@@ -1,37 +1,55 @@
 # Stability Checker
 
 ## Overview
-
-The Stability Checker runs the `testing-kyma.sh` script in a loop, and reports the results on a Slack channel.
+The purpose of the Stability Checker is to check if a cluster is stable. 
+To ensure the cluster's stability, install the Stability Checker in the cluster. 
+It runs a testing script in a loop and sends notifications with the test executions summary on a Slack channel.
 
 ## Installation
+Install the Stability Checker on the Kyma cluster as a Helm chart. Find the chart definition in the `deploy/chart` directory.
+1. Configure kubectl and helm to point your cluster by specifying **KUBECONFIG** environment variable. 
+```
+export KUBECONFIG="/{path to kubeconfig}"
+```
 
-You can install the Stability Checker on the Kyma cluster as a chart. Find the chart definition in the `deploy/chart` directory.
-
-> **NOTE:** You must install the chart after running the core tests, to avoid running the same tests in parallel.
-
-## Usage
+> **NOTE:** Ensure that kubeconfig allows you to update cluster state by installing helm charts. 
+2. Provision volume with a testing script, which then will be used by the Stability Checker. You can use 
+`local/provision_volume.sh` script. The script copies all files placed in `local/input` directory to PV.
 
-Stability Checker does not contain testing scripts. The chart value `.Values.pathToTestingScript` defines which script the system runs.
-Ensure you have the file available in a persistent volume defined as `.Values.storage.claimName`, which is mounted as a `data` directory in the Stability Checker Pod.
+3. Install the Stability Checker as a Helm chart and provide the proper configuration:
 
-To simulate the process of providing scripts, see the `local/provision_volume.sh` script, which populates the volume with files from the `local/input` directory.
+```
+helm install deploy/chart/stability-checker \
+  --set slackClientWebhookUrl="" \
+  ...
+  --namespace="kyma-system" \
+  --name="stability-checker"
 
-### Deliver the chart
+```
 
-Download the gzipped chart from:
+Below you can find configuration options:
 
-`https://github.com/kyma-project/stability-checker/raw/{branchName}/deploy/chart/stability-checker-0.1.0.tgz`
+ | Name | Default value | Description |
+ |------|---------------|-------------|
+**storage.claimName** |`stability-test-scripts-pvc`| Name of the Persistent Volume Claim (PVC) which is attached to the Stability Checker Pod. The volume is visible in the Pod under the `/data` path. 
+**pathToTestingScript** |`/data/input/testing-kyma.sh`| Full path to the testing script. As the script is delivered inside the PVC, the path must start with `/data`.
+**slackClientWebhookUrl** |-| Slack client webhook URL. For details, see the configuration of integrated applications in the Slack Workspace. 
```

Comment:
I've searched through the Slack Workspace and couldn't find it :) Let's have a discussion on how to write this to leave no doubt.